gui/src/components/preboarding/login.vue

```diff
@@ -86,110 +86,15 @@
-              />
-            </label>
-            <input
-              class="cortx-form__input_text"
-              type="password"
-              id="user-password"
-              name="user-password"
-              v-model.trim="resetAccountForm.password"
-              @input="$v.resetAccountForm.password.$touch"
-            />
-            <div class="cortx-form-group-label cortx-form-group-error-msg">
-              <label
-                id="new-password-required-error"
-                v-if="
-                  $v.resetAccountForm.password.$dirty &&
-                  !$v.resetAccountForm.password.required
-                "
-                >{{ $t("common.password-required") }}</label
-              >
-              <label
-                id="new-password-invalid-error"
-                v-else-if="
-                  $v.resetAccountForm.password.$dirty &&
-                  !$v.resetAccountForm.password.passwordRegex
-                "
-                >{{ $t("common.invalid-password") }}</label
-              >
-            </div>
-          </div>
-        </v-col>
-        <v-col class="col-6 ml-7 pt-0">
-          <div
-            class="cortx-form-group"
-            :class="{
-              'cortx-form-group--error':
-                $v.resetAccountForm.confirmPassword.$error
-            }"
-          >
-            <label
-              class="cortx-form-group-label"
-              for="confirm-password"
-              id="confirm-password-label"
-              >{{ $t("common.confirm-password-label") }}</label
-            >
-            <input
-              class="cortx-form__input_text"
-              type="password"
-              id="confirm-password"
-              name="confirm-password"
-              v-model.trim="resetAccountForm.confirmPassword"
-              @input="$v.resetAccountForm.confirmPassword.$touch"
-            />
-            <span
-              id="confirm-password-notmatch-error"
-              class="cortx-form-group-label cortx-form-group-error-msg"
-              v-if="
-                $v.resetAccountForm.confirmPassword.$dirty &&
-                !$v.resetAccountForm.confirmPassword.sameAsPassword
-              "
-              >{{ $t("common.password-not-match") }}</span
-            >
-          </div>
-        </v-col>
-        <v-col class="col-6 ml-7 pb-6 pt-0">
-          <button
-            type="button"
-            id="reset-password-button"
-            class="cortx-btn-primary"
-            @click="resetPassword()"
-            :disabled="$v.resetAccountForm.$invalid"
-          >
-            {{ $t("login.reset-btn") }}
-          </button>
-        </v-col>
-      </v-card>
-    </v-dialog>
-      
+
+    <cortx-reset-password
+      v-if="showResetPasswordDialog"
+      :showResetPasswordDialog="showResetPasswordDialog"
+      :authToken="authToken"
+      :username="loginForm.username"
+      @closeResetPass="closeResetPassword"
```

Comment:
Update @closeResetPass="closeResetPassword" to @complete="onResetPasswordCompletion". This will receive a parameter resetPasswordCompletionStatus ({ status: "success", message: "Success msg if any"} or { status: "failed", message: "err msg"}). You have to custom error handle using try/catch.

Reply:
Updated method name